src/Button.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Button({ children, onClick }) {
+  return (
+    <button type="button" onClick={() => onClick(children)}>
```

Comment:
```suggestion
    <button type="button" onClick={onClick}>
```

onClick의 인자로 children을 넘겨줄 이유가 있을까요?

단순하게 버튼 컴포넌트에서는 버튼 클릭이 있을때 넘겨받은 클릭 함수만 실행해주도록 해도 될 것 같아요. 
children을 넘기게 되면 우리가 의도치 않은 버튼 컴포넌트를 사용하는 쪽에서 넘기는 값을 전달하게되어 사이드 이펙트가 발생할 수 있을 것 같아요.